server/handlers/load_test_handler.go

```diff
@@ -364,22 +364,40 @@ func (h *Handler) executeLoadTest(ctx context.Context, req *http.Request, profil
 	resultsMap["load-generator"] = loadTestOptions.LoadGenerator
 
 	mk8sContexts, ok := req.Context().Value(models.KubeClustersKey).([]models.K8sContext)
-	if !ok || len(mk8sContexts) == 0 {
-		h.log.Error(ErrInvalidK8SConfig)
-		return
+	if !ok {
+		h.log.Warn(ErrInvalidK8SConfig)
 	}
 	var wg sync.WaitGroup
+	h.log.Info(mk8sContexts)
```

Comment:
Does this log any secrets by any chance? Ignore if it does not

Reply:
It will. Thanks for pointing that out, we don't need that log anyways.

Reply:
Thanks, @hershd23 👏